thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/anomaly/monitor/MonitorTaskRunner.java

```diff
@@ -108,6 +111,28 @@ private void executeMonitorUpdate(MonitorTaskInfo monitorTaskInfo) {
     }
   }
 
+  private int cleanUpMergedAnomalies(int cleanUpDays) {
+    DateTime cleanupStartDate = new DateTime().withTimeAtStartOfDay().minusDays(cleanUpDays);
+    DateTime cleanupEndDate = cleanupStartDate.plusDays(1);
```

Comment:
Looks like we are cleaning up the anomalies in the 90th-day before today. What about the anomalies before that?

Reply:
The clean window is 1 day and in a rolling window fashion. The anomalies before that should already been cleaned.